esp-box-3/include/bsp/esp-box-3.h

```diff
@@ -255,6 +266,47 @@ esp_err_t bsp_spiffs_mount(void);
  */
 esp_err_t bsp_spiffs_unmount(void);
 
+/**************************************************************************************************
+ *
+ * uSD card
+ *
+ * After mounting the uSD card, it can be accessed with stdio functions ie.:
+ * \code{.c}
+ * FILE* f = fopen(BSP_MOUNT_POINT"/hello.txt", "w");
+ * fprintf(f, "Hello %s!\n", bsp_sdcard->cid.name);
+ * fclose(f);
+ * \endcode
+ *
+ * @attention IO2 is also routed to RGB LED and push button
+ **************************************************************************************************/
+#define BSP_MOUNT_POINT      CONFIG_BSP_uSD_MOUNT_POINT
```

Comment:
```suggestion
#define BSP_MOUNT_POINT      CONFIG_BSP_SD_MOUNT_POINT
```

I don't like the small letter here and we are using the same name in all BSPs.